src/Utility/Muted.js

```diff
@@ -1,17 +1,80 @@
 const BotModule = require('../BotModule');
 const RoleDeterminer = require('../Helper/RoleDeterminer');
 
+const Message = {
+    autoMuted: 'Muted @{username}#{discriminator} automatically upon rejoin',
+    unableToAutoMute: 'I was unable to automatically mute @{username}#{discriminator}. My permissions may be messed up. Please contact a developer immediately.\\n**Error: ** ```{error}```\''
+}
+
 module.exports = BotModule.extend({
     config: null,
     dependencies: {
-        'config': 'config'
+        'config': 'config',
+        'muteRepository': 'muteRepository'
     },
     initialize: function (dependencyGraph) {
         this._super(dependencyGraph);
+
+        this.discordClient.on('ready', this.loadMutes.bind(this));
```

Comment:
Is `this.loadMutes.bind(this)` necessary? Can't you simply write down `this.loadMutes` here? The `.on`-function requires a function as the second argument that has the required signature. I know that `this` in JS can be a bit confusing from time to time.

Reply:
Apparently not. I added the binds because I got an error regarding accessing .guilds from undefined.